src/table/table.scss

```diff
@@ -16,75 +15,203 @@
     box-sizing: border-box;
   }
 
-  > .iui-tables-body {
-    width: 100%;
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
+  .iui-header {
+    @include iui-table-header;
+  }
+
+  .iui-body {
```

Comment:
This should be `.iui-table-body` to avoid conflicts with the global body class

https://github.com/iTwin/iTwinUI/blob/3fb2a6bd60eb936a1d79b8c8adf24ded0f6391c0/src/style/global.scss#L5